meta/main.yml

```diff
@@ -52,3 +54,5 @@ dependencies:
     name: numactl
   - src: https://github.com/cisagov/ansible-role-pip
```

Comment:
Please add [cisagov/ansible-role-python](https://github.com/cisagov/ansible-role-python) as a dependency since we are installing Python packages with `pip`.

Reply:
👍 Please see https://github.com/cisagov/ansible-role-mongo/pull/12/commits/71ed25bd6a9e432678af68a5d760eb09d1e7dd73.